documentation/week3/src/components/Form.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+
+export default class Form extends Component {
+
+    render() {
```

Comment:
Use this `const {discription, deadline} = this.props;` instead of writing `this.props.whatever` multiple times. **This is better performance wise and also code readability**